### PR TITLE
CodeCamp #101: Support MMDetection 3.x RTMDet model deployment on RV1126

### PR DESCRIPTION
**Pull request by Qingrenn**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily receiving feedbacks. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Support MMDetection 3.x RTMDet model deployment on RV1126.

## Modification

1. Partition RTMDet ONNX representation.
2. Add RTMDet deploy config.
3. Add RTMDet postprocess in sdk.

## BC-breaking (Optional)

Does the modification introduce changes that break the backward-compatibility of the downstream repositories?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

1. Convert RTMDet 
```bash
cd ${MMDeploy}

python ${MMDeploy}/tools/deploy.py \
${MMDeploy}/configs/mmdet/detection/detection_rknn-int8_static-640x640.py \
${MMDet}/configs/rtmdet/rtmdet_m_8xb32-300e_coco.py \
./checkpoints/rtmdet_m_8xb32-300e_coco_20220719_112220-229f527c.pth \
${MMDeploy}/demo/resources/det.jpg \
--work-dir ./mmdeploy_models/mmdet/rtmdet_m \
--device cpu \
--dump-info
```

2. Compile mmdeploy sdk.
3. Push sdk, image and rtmdet_m to rv1126.
4. Perform infer in rv1126.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
5. The modification is covered by complete unit tests. If not, please add more unit tests to ensure the correctness.
6. If the modification has a dependency on downstream projects of a newer version, this PR should be tested with all supported versions of downstream projects.
7. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

File: mmdeploy/codebase/mmdet/deploy/object_detection_model.py [hunk 1]
```diff
@@ -755,6 +755,14 @@ def _get_bboxes(self, outputs: List[Tensor], metainfos: Any):
                 batch_img_metas=metainfos,
                 cfg=self.model_cfg._cfg_dict.model.test_cfg,
                 rescale=True)
+        elif head_cfg.type == 'RTMDetSepBNHead':
+            divisor = round(len(outputs) / 2)
```
Comment (AllentDan) [inline mmdeploy/codebase/mmdet/deploy/object_detection_model.py:759]: what if objectness is not None?

**Review by AllentDan — COMMENTED** [review]
Hi, @Qingrenn. Could you rebase the branch to dev-1.x. My commits are now in the PR.

File: csrc/mmdeploy/preprocess/transform/normalize.cpp [hunk 2]
```diff
@@ -99,13 +99,14 @@ class Normalize : public Transform {
       Tensor dst;
       if (to_float_) {
         OUTCOME_TRY(normalize_.Apply(tensor, dst));
+        data[key] = std::move(dst);
```
Comment (lvhan028) [inline csrc/mmdeploy/preprocess/transform/normalize.cpp:102]: There is already `data[key] = std::move(dst);` after the if-else condition branch

**Review by AllentDan — COMMENTED** [review]
May fix the lint error. If you are using vscode, may install *clang-format* plugin and set property.json like following:
```
    "editor.formatOnSave": true,
    "clang-format.fallbackStyle": "Google",
    "clang-format.language.cuda.fallbackStyle": "Google",
    "clang-format.language.cuda.style": "Google",
    "clang-format.language.cpp.enable": true,
    "[cpp]": {
        "editor.defaultFormatter": "xaver.clang-format",
        "editor.wordBasedSuggestions": false,
        "editor.suggest.insertMode": "replace",
        "editor.semanticHighlighting.enabled": true
    },
    "clang-format.executable": "/home/linuxbrew/.linuxbrew/bin/clang-format",
```

**Review by AllentDan — COMMENTED** [review]
May update docs like [here](https://github.com/open-mmlab/mmdeploy/blob/dev-1.x/docs/en/01-how-to-build/rockchip.md#troubleshooting)?

**Review by AllentDan — APPROVED** [review]
LGTM

**Comment by Qingrenn** [comment]
There is a url will be valid after these codes being merged, but this will inevitably lead to a lint error. 
Specifically, I add a config file `detection_rknn-int8_static-640x640.py`, and the corresponding url (https://github.com/open-mmlab/mmdeploy/blob/dev-1.x/configs/mmdet/detection/detection_rknn-int8_static-640x640.py) will be valid after this part of code being merged.

**Pull request merged**